### PR TITLE
[JBIDE-19506] added tests for Connection

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

File: plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/common/core/connection/IConnection.java [hunk 1]
```diff
@@ -22,7 +22,13 @@
 	public static final String PROPERTY_USERNAME = "username";
 	public static final String PROPERTY_PASSWORD = "password";
 	public static final String PROPERTY_REMEMBER_PASSWORD = "rememberPassword";
+	public static final String PROPERTY_AUTHSCHEME = "authtype";
+	public static final String PROPERTY_REMEMBER_TOKEN = "rememberToken";
 
+	public static final String SECURE_STORAGE_BASEKEY = "org.jboss.tools.openshift.core";
+	public static final String SECURE_STORAGE_PASSWORD_KEY = "password";
+	public static final String SECURE_STORAGE_TOKEN_KEY = "token";
```
Comment (fbricon) [inline plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/common/core/connection/IConnection.java]: these are implementation details, specific to v3. don't belong here

**Comment by fbricon** [comment]
except for the moving of internal keys to the public API, that should be reverted, rest looks good to me

**Comment by adietish** [comment]
testPR

**Comment by adietish** [comment]
testPR

**Comment by adietish** [comment]
testPR

**Pull request merged**